src/shared/components/Header/Menu/component.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+
+import { MenuWrapper, MenuListWrapper } from './styles';
+
+type MenuProps = {
+  isOpenMenu: boolean;
+};
+
+type MenuItem = {
+  name: string;
+  link: string;
+};
+
+type MenuList = {
```

Comment:
let's move this MenuList and MenuItem to the shared types - and import and use it here and in footer.

Reply:
Done!